devices.php

```diff
@@ -910,6 +910,9 @@ function buildCustomAttributes($template, $device) {
 					print "\n\t<option value=\"$dcaValue\"$selected>$dcaValue</option>";
 				}
 				echo '</select></div>';
+			} else if ($cvtype=="url") {
+				echo '<div><input type="text"',$validation,' name="',$inputname,'" id="',$inputname,'" value="',$customdata["value"],'">';
+				echo '<button type="button" onclick=window.location.href="',$customdata["value"],'" value="open">Open</button></div>';

```

Comment:
OK thanks for the feedback. Let me have another crack at it tomorrow night (Australian time).